common/src/main/java/com/wynntils/features/commands/LootrunFeature.java

```diff
@@ -2,16 +2,19 @@
  * Copyright © Wynntils 2022.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.features;
+package com.wynntils.features.commands;
 
 import com.wynntils.core.components.Models;
+import com.wynntils.core.config.Category;
 import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigCategory;
 import com.wynntils.core.config.ConfigHolder;
 import com.wynntils.core.config.RegisterConfig;
 import com.wynntils.core.features.Feature;
 import com.wynntils.utils.colors.CommonColors;
 import com.wynntils.utils.colors.CustomColor;
 
+@ConfigCategory(Category.COMMANDS)
```

Comment:
This is definitely wrong. Lootrun support is a major new functionality. Just because there are commands you can use to control it does not mean it belongs here.That category is for command aliases, command expansion etc.